kafka/producer/simple.py

```diff
@@ -41,19 +42,24 @@ class SimpleProducer(Producer):
     def __init__(self, client, async=False,
                  req_acks=Producer.ACK_AFTER_LOCAL_WRITE,
                  ack_timeout=Producer.DEFAULT_ACK_TIMEOUT,
-                 codec=None,
+                 codec=CODEC_NONE,
                  batch_send=False,
                  batch_send_every_n=BATCH_SEND_MSG_COUNT,
                  batch_send_every_t=BATCH_SEND_DEFAULT_INTERVAL,
-                 random_start=True):
+                 random_start=True,
+                 maxsize=0):
         self.partition_cycles = {}
         self.random_start = random_start
         super(SimpleProducer, self).__init__(client, async, req_acks,
                                              ack_timeout, codec, batch_send,
                                              batch_send_every_n,
-                                             batch_send_every_t)
+                                             batch_send_every_t,
+                                             maxsize)
 
     def _next_partition(self, topic):
+        # for each message sent a partition is chosen, if random_start is True
+        # a random partition is selected, otherwise the next partition in the cycle
+        # is picked up
```

Comment:
Along with bc comment, I think we can leave this comment out, we can reduce the size of the diff.